### DISCUSSION TITLE
DSEP Working Group - Call 1 - Feb 16th, 2022

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/DSEP-Specification#16] [Announcements]
Today we had the first official DSEP Working Group Call at 4 pm. 

![Call 1 - Agenda](https://user-images.githubusercontent.com/52468749/154258102-d588e5ba-0b10-4c87-a2dd-1d913bcf8a5e.png)

**Comment by sujitrs** [comment]
**Call Deck and Recording**  [DSEP Working Group - Call 1 - Feb 16th, 2022.pdf](https://github.com/beckn/DSEP-Specification/files/8080331/DSEP.Working.Group.-.Call.1.-.Feb.16th.2022.pdf)

**Action Items** ![image](https://user-images.githubusercontent.com/23429265/154277817-818312e8-068f-44f9-8306-cbc808b31ea1.png)

**Reply by sujitrs** [reply]
Use case Form https://forms.gle/KJ1gG9Mj7R9jnHzJ9
